src/fsharp/CompileOps.fs

```diff
@@ -2618,16 +2617,8 @@ type TcConfig private (data : TcConfigBuilder,validate:bool) =
         | Some x -> 
             [tcConfig.MakePathAbsolute x]
         | None -> 
-            // When running on Mono we lead everyone to believe we're doing .NET 2.0 compilation 
-            // by default. 
             if runningOnMono then 
-                let mono10SysDir = System.Runtime.InteropServices.RuntimeEnvironment.GetRuntimeDirectory() 
```

Comment:
This code is only relevant to Mono so was always ignored by Visual F# Tools.  Replacing it here by the version that is in fsharp/fsharp for uniformity and alignment.